src/pages/RepoPage/FailedTestsTab/CodecovCLI/CodecovCLI.tsx

```diff
@@ -0,0 +1,225 @@
+  )
+}
+
+const Step4Script = `codecovcli upload-process`
+
+function Step4() {
+  return (
+    <Card>
+      <Card.Header>
+        <Card.Title size="base">
+          Step 4: Upload coverage to Codecov using the CLI
+        </Card.Title>
+      </Card.Header>
+      <Card.Content className="flex flex-col gap-4">
+        <p>
+          Codecov offers existing wrappers for the CLI (Github Actions, Circle
+          CI Orb, Bitrise Step) that makes uploading coverage to Codecov easy,
+          as described{' '}
+          <A
+            to={{ pageName: 'uploaderCLI' }}
+            isExternal={true}
+            hook="failed-tests-onboarding"
+          >
+            here
+          </A>
+          .
+        </p>
+        <p>
+          If you&apos;re running a different CI, you can upload coverage as
+          follows
+        </p>
+        <CodeSnippet clipboard={Step4Script}>{Step4Script}</CodeSnippet>
+        <p>
+          Go ahead and merge these changes in. In the next step we&apos;ll
+          verify if things are working correctly.
+        </p>
+      </Card.Content>
+    </Card>
+  )
+}
+
+function Step5() {
+  const [isExpanded, setIsExpanded] = useState(false)
+
+  return (
+    <div>
+      <Card>
+        <Card.Header>
+          <Card.Title size="base">Step 3: Run your test suite</Card.Title>
+        </Card.Header>
+        <Card.Content className="flex flex-col gap-4">
+          <p>
+            You can inspect the workflow logs to see if the call to Codecov
+            succeeded.
+          </p>
+          <img src={testsRunning.toString()} alt="CLI tests" />
+          <p>
+            Run your tests as usual. You need to have some failed tests to view
+            the failed tests report.
+          </p>
+        </Card.Content>
+      </Card>
+      <ExpandableSection className="mt-[-1px]">
+        <ExpandableSection.Trigger
+          isExpanded={isExpanded}
+          onClick={() => setIsExpanded(!isExpanded)}
+        >
+          <p className="font-normal">
+            Here are examples of failed test reports in PR comments. Comment
+            generation may take time.
+          </p>
+        </ExpandableSection.Trigger>
+        <ExpandableSection.Content>
+          <img
+            src={testsPRComment.toString()}
+            alt="Tests in PR comment"
+            className="w-full"
+          />
+        </ExpandableSection.Content>
+      </ExpandableSection>
+    </div>
+  )
+}
+
+function Step6() {
+  return (
+    <Card>
+      <Card.Header>
+        <Card.Title size="base">Step 4: View results and insights</Card.Title>
+      </Card.Header>
+      <Card.Content className="flex flex-col gap-1">
+        <p>
+          After the test run completion, you&apos;ll be able to see the failed
+          tests result on the following areas:
+        </p>
+        <ul className="list-inside list-disc">
+          <li>Github pull request comment</li>
+          <li>Failed tests dashboard here.</li>
```

Comment:
should "here" be a link to something?

Reply:
Nah here means in this route instead of the onboarding page

Reply:
ahh okay for sure